scripts/platforms/wellfound_apply.py

```
@@ -0,0 +1,93 @@
+
+                await page.locator(selector).first.click()
+
+                clicked = True
+
+                print(
+                    f"Clicked: {selector}"
+                )
+
+                break
+
+            except:
```

Ruff (0.15.12)

[error] Do not use bare `except`

(E722)